.changelog/unreleased/improvements/2128-marker-nav-docs.md

```diff
@@ -0,0 +1 @@
+* Address missing documentation on marker nav command [#2134](https://github.com/provenance-io/provenance/issues/2128).
```

Comment:
No thanks... the included succinct message is appropriate since it matches the issue title and provides a link to the specified issue for further details.